pulpcore/pulpcore/app/viewsets/repository.py

```diff
@@ -177,6 +178,8 @@ class RepositoryVersionViewSet(NamedModelViewSet,
     serializer_class = RepositoryVersionSerializer
     queryset = RepositoryVersion.objects.exclude(complete=False)
     filter_class = RepositoryVersionFilter
+    filter_backends = (OrderingFilter,)
+    ordering = ('-number',)
```

Comment:
I feel like this should count up and not down. But then again I guess if this is eventually paginated, you'd want to see the latest ones first?